exercises/beer-song/BeerSongTest.swift

```diff
@@ -0,0 +1,24 @@
+        XCTAssertEqual(expectedBeerSongVerses, beerSong.generateVersesOfBeerSong())
+    }
+
+    override func tearDown() {
+        beerSong = nil
```

Comment:
ARC (automatic reference count ) takes care of this for you.  Maybe I am missing something?